libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/hwdef.dat

```diff
@@ -43,6 +43,16 @@ PC9  TIM8_CH4 TIM8 PWM(8) GPIO(57)
 PB11 VTX_PWR OUTPUT HIGH GPIO(81)
 define RELAY2_PIN_DEFAULT 81
 
+# second battery sensor on second ESC connector
+PA1 BATT2_VOLTAGE_SENS ADC1 SCALE(1)
+PC4 BATT2_CURRENT_SENS ADC1 SCALE(1)
+
+# second battery setup (note external current sensor required)
```

Comment:
has to be in defaults.parm

Reply:
I added support for the second battery in hwdef